Website/src/rendering/src/pages/shop/checkout/cart.tsx

```diff
@@ -11,19 +16,32 @@ interface CartProps {
 }
 
 const Cart = (props: CartProps): JSX.Element => {
```

Comment:
I believe the cart props are not used. Could they be removed with their interface?